src/django/pfb_network_connectivity/models.py

```diff
@@ -5,7 +5,7 @@
 from __future__ import unicode_literals
 import uuid
 
-from django.db import models
+from django.contrib.gis.db import models
```

Comment:
This seemed easiest, no issues with swapping the base class. Other option was to have two PFBModel base classes, one inheriting from gis model and one from the regular one, but that was way more effort for essentially no gain.